addons/mail/controllers/mail_template.py

```diff
@@ -0,0 +1,24 @@
+from odoo import http
+from odoo.http import request
+from odoo.osv import expression
+
+
+class MailTemplateController(http.Controller):
+    @http.route('/mail/template', type='json', auth='user')
```

Comment:
Maybe something like
```suggestion
    @http.route('/mail/template/fetch_usable', type='json', auth='user')
```